models/attachment.go

```diff
@@ -104,7 +114,7 @@ func getAttachmentByUUID(e Engine, uuid string) (*Attachment, error) {
 	attach := &Attachment{UUID: uuid}
 	has, err := e.Get(attach)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
I'm that fucking retarded who asked but I did ask to do that in completly different file and before you rewrote error handling logic....